### ISSUE TITLE
Nginx EE Fail

### ISSUE THREAD

**Issue by wiroy** [VirtuBox/ubuntu-nginx-web-server#12]
Hello

During the installation nginx ee is giving nginx fail error in the last stage please take care.

**Comment by VirtuBox** [comment]
Hello @wiroy,

can you post nginx-ee install log available in /tmp/nginx-ee.log ?

**Comment by wiroy** [comment]
Hello @VirtuBox 

Nginx ee log : https://paste.ubuntu.com/p/Q2kTDhB6sJ/

**Comment by wiroy** [comment]
yes there is a naxsi waf installation error and updating of the existing optimizations is needed. @VirtuBox @nsgoyat
